efficientdet/keras/train_lib.py

```diff
@@ -562,7 +562,7 @@ def test_step(self, data):
     if 'object_detection' in self.config.heads:
       det_loss = (
           self._detection_loss(cls_outputs, box_outputs, labels, loss_vals))
-      total_loss += det_loss
+      total_loss += det_loss[0]
```

Comment:
It should be ```tf.math.reduce_sum(det_loss)``` and keep total loss same as train step.

Reply:
However, in `_detection_loss`:
https://github.com/google/automl/blob/3958f1b68a02b5e67075e6c45ecc2376131f4165/efficientdet/keras/train_lib.py#L478-L480
which means the first term in the return value is already the weighted sum of the rest terms. Wouldn't applying `tf.math.reduce_sum` make the summation duplicate?

Reply:
Yes, you're right. Change total_loss += det_loss[0] too.

Reply:
I prefer to change _detection_loss function:
```return total_loss```

Reply:
Modified as you suggested.